bal/bcnode.py

```diff
@@ -96,45 +97,60 @@ def call(self, command, silent= False, data=''):
                                            cdir=self.cdir,
                                            sdir=self.sdir)
         else:
-            cmd += " "  + command
+            cmd += " " + command
         if silent:
-            result = self.cmd( cmd )
+            result = self.cmd(cmd)
         else:
-            result = self.cmdPrint( cmd )
+            result = self.cmdPrint(cmd)
 
         debug("command: %s = %s" % (cmd, result))
         return result
 
+
 class POWNode(BCNode):
     """A POWNode is a BCNode that is running an POWBlockchain."""
 
-    def __init__( self, name, bcclass=None, inNamespace=True,
-                  server='blockchain.py',
-                  sargs='-p {port} -s {socket} -d 2 -k {sdir}/{IP}pow.pem -n {name} -sp {simulation_path}',
-                  sdir='/tmp/bcn',
-                  client='curl',
-                  cargs="-s -X {method} http://{IP}:{port}/{command}",
-                  cdir=None,
-                  ip="127.0.0.1", port='5000', **params ):
+    def __init__(
+            self,
+            name,
+            bcclass=None,
+            inNamespace=True,
+            server='blockchain.py',
+            sargs='-p {port} -s {socket} -d 2 -k {sdir}/{IP}pow.pem -n {name} -sp {simulation_path}',
+            sdir='/tmp/bcn',
+            client='curl',
+            cargs="-s -X {method} http://{IP}:{port}/{command}",
+            cdir=None,
+            ip="127.0.0.1",
+            port='5000',
+            **params):
```

Comment:
I prefer these one to be compact as possible, to increase the readability. One line for each parameter kinda makes the code longer than it should be.